tasks/main.yml

```diff
@@ -1,11 +1,16 @@
 ---
 
-- include: install.yml
+# Include variables and define needed variables.
+- name: Include OS-specific variables.
```

Comment:
Should probably conditionally set the variables here, as we did in Karaf, so they can be overridden. 

Maybe some comments in the defaults as well... 

I know the Karaf stuff just went in this afternoon, just wanted to make a note here so it doesn't get forgotten.

Reply:
done

Reply:
done, see below.